modules/jfx.incubator.richtext/src/main/java/jfx/incubator/scene/control/richtext/model/StyledTextModel.java

```diff
@@ -135,7 +137,7 @@ public interface Listener {
     /**
      * Returns the plain text string for the specified paragraph.  The returned text string cannot be null
      * and must not contain any control characters other than TAB.
-     * The caller should never attempt to ask for a paragraph outside of the valid range.
+     * The caller must not attempt to ask for a paragraph outside of the valid range.
```

Comment:
It might be worth adding the note about "doing otherwise might result in an exception..." as you mentioned below.